### DISCUSSION TITLE
How to deal with Continuous Collision Detection?

### DISCUSSION THREAD

**Discussion by joao-pm-santos96** [stack-of-tasks/pinocchio#2185] [Q&A]
Hi guys! 
I have a quick question, does pinocchio offer Continuous Collision Detection capabilities? 
If not, how do you suggest we approach this matter?
Thank you!

**Comment by florent-lamiraux** [comment]
Hello @joao-pm-santos96 .
Continuous collision detection is implemented in HPP https://humanoid-path-planner.github.io/hpp-doc/, a path planning library built on top of pinocchio.

**Reply by joao-pm-santos96** [reply]
Awesome! But I can't find it... can you please point me to an example or documentation page?

**Reply by florent-lamiraux** [reply]
Following the link provided above, click on "get started now" and choose which installation you prefer binary packages (ubuntu) or compilation from source. Then there is a package [hpp_tutorial](https://github.com/humanoid-path-planner/hpp_tutorial) that you can install to start.
